src/components/v2/chartDeploymentHistory/DockerListModal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { ReactComponent as Close } from '../assets/icons/ic-close.svg'
+import docker from '../../../assets/icons/misc/docker.svg'
+
+export default function DockerListModal({ closeTab, dockerList }) {
+    return (
+        <div className="docker-wrapper" onClick={(e) => e.stopPropagation()}>
```

Comment:
Why do we need this?

Reply:
Removing this & adding esc button event as per discussion with Utkarsh